products/alp/main.pm

```diff
@@ -42,7 +42,7 @@ sub load_common_tests {
     loadtest 'microos/image_checks';
     loadtest 'microos/one_line_checks';
     loadtest 'microos/services_enabled';
-    loadtest 'microos/cockpit_service';
```

Comment:
Why this removal? (just asking)

Reply:
I just checked ALP-VM.x86_64-0.0.1-kvm-Build9.2, cockpit_service failed, please let me also well, so I can update my yaml schedule :)

Reply:
I didn't really check why it is failing. Fell free to do so.